benchmarks/templates/benchmarks/table.html

```diff
@@ -15,17 +15,17 @@
                     <strong class="want_to_click" style="padding: 5px; display: {{ benchmark_parents|in_set_hidden:benchmark.name }}">{{ benchmark.parent }}:</strong>{% endif %}
                     
                 {% if benchmark.link %}
-                    <a href="{{ benchmark.link }}">{{ benchmark.name }}
-                        {%  if benchmark.version %}
-                        <span class="benchmark-version">v{{ benchmark.version }}</span>
-                        {% endif %}
-                    </a></p>
```

Comment:
not sure if this is a merge error or intentional, but I think we should leave the version in there (#12)